src/Voron/Impl/Journal/WriteAheadJournal.cs

```diff
@@ -749,17 +747,21 @@ public void ApplyLogsToDataFile(CancellationToken token, TimeSpan timeToWait)
 
                         try
                         {
-                            UpdateJournalStateUnderWriteTransactionLock(txw, lastProcessedJournal, lastFlushedTransactionId, unusedJournals);
+                            UpdateJournalStateUnderWriteTransactionLock(txw, journalSnapshots, lastProcessedJournal, lastFlushedTransactionId);
+
+                            if (_waj._logger.IsInfoEnabled)
+                                _waj._logger.Info($"Updated journal state under write tx lock (txId: {txw.Id}) after waiting for {sp.Elapsed}");
                         }
                         catch (Exception e)
                         {
+                            if (_waj._logger.IsOperationsEnabled)
+                                _waj._logger.Operations($"Failed to update journal state under write tx lock (waited - {sp.Elapsed})");
```

Comment:
Missing the actual exception in the logger.

Reply:
Fixed in https://github.com/ravendb/ravendb/pull/11972